ledger/sync/src/scp_network_state.rs

```diff
@@ -28,18 +25,14 @@ pub struct SCPNetworkState<ID: GenericNodeId = NodeID> {
 
     // Highest slot that a given node has externalized.
     id_to_current_slot: HashMap<ID, SlotIndex>,
-
-    // Logger.
-    logger: Logger,
 }
 
 impl<ID: GenericNodeId + Clone + Eq + PartialEq + Hash> SCPNetworkState<ID> {
-    pub fn new(local_node_id: ID, local_quorum_set: QuorumSet<ID>, logger: Logger) -> Self {
+    pub fn new(local_node_id: ID, local_quorum_set: QuorumSet<ID>) -> Self {
```

Comment:
the equivalent to this in the crate tests `NodeConfig` includes a "name" and a hashset of peers that receive our broadcasts -- should we support broadcasting to peers that are not in quorum? Seems like it could be useful down the road for redundant configurations, auditing nodes, etc...